doc/ksdomain.py

```diff
@@ -190,11 +197,19 @@ class KOSMethod(KOSObject):
     def handle_signature(self, sig, signode):
         m = ks_sig_re.match(sig)
         name = m.group('object')
+        struct = None  #might override further down
 
         current_struct = self.env.temp_data.get('ks:structure')
         if m.group('prefix') is None:
             if current_struct is not None:
+                struct = current_struct
```

Comment:
This is the line that was missing when this got copied from KOSAttribute above.  It's relevant because without it when you make a tag like ``.. method:: methname``, where the ``structname:`` prefix is implied rather than explicitly stated, the presumed default prefix wasn't being added due to this missing line.